.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,95 @@
+name: 🐞 Bug Report
+description: Report a bug in Sentry for Godot
+labels:
+  - Bug
+  - Godot
+body:
+  - type: markdown
+    attributes:
+      value: |
+        - Write a descriptive title above.
+
+  - type: input
+    attributes:
+      label: How do you use Sentry?
+      description: Sentry SaaS (sentry.io) or self-hosted/on-premise (which version?)
+      placeholder: Sentry SaaS
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Sentry SDK version
+      description: |
+        - Specify Sentry SDK version.
+        - You can check the version in the Godot Output panel or in the log file.
+      placeholder: 0.6.0
+    validations:
+      required: true
+
+  - type: dropdown
+    id: variant
+    attributes:
+      label: How did you install the SDK?
+      options:
+        - GitHub release
+        - AssetLib
```

Comment:
we're not there yet, right?

any news about their re-write? I wonder if it's time for us to automate uploading it